Makefile

```diff
@@ -37,10 +37,10 @@ src/pykcs11_wrap.cpp-py3: src/pykcs11.i
 src/pykcs11.i: src/opensc/pkcs11.h src/pkcs11lib.h src/pykcs11string.h src/ck_attribute_smart.h
 	touch $@
 
-dist: clean
+dist: src/pykcs11_wrap.cpp-py2 src/pykcs11_wrap.cpp-py3 clean
```

Comment:
Yep, you're right! And even worse, two parallel SWIG runs stuck with an error — they are trying to overwrite each other's LowLevel.py. I'm working on that…